client/python/gradio_client/client.py

```diff
@@ -113,7 +113,7 @@ def __init__(
             self.headers.update(headers)
         self.ssl_verify = ssl_verify
         self.space_id = None
-        self.cookies: dict[str, str] = {}
+        self.cookies: dict[str, str] = dict(httpx_kwargs.get('cookies', {}))
```

Comment:
I think you'll need to pop, otherwise will get an error later on when all the kwargs in `httpx_kwargs` are passed directly and also `cookies` is passed as an explicit parameter
```suggestion
        self.cookies: dict[str, str] = dict(httpx_kwargs.pop('cookies', {}))
```